src/ploneintranet/search/interfaces.py

```diff
@@ -0,0 +1,61 @@
+""" Interfaces for search API
+"""
+from zope.interface import Interface, Attribute
+
+from . import _
+
+
+class IGlobalSearch(Interface):
```

Comment:
'Search' was too broad a term to describe what we are doing here. This is designed to represent the user stories for a global / site-wide search, so we tried to get that into the name.